src/utils/handleAndSignImages.ts

```diff
@@ -13,7 +13,7 @@ export function handleAndSignImages(
   imageUrl?: string;
   lowResUrl?: string;
 } {
-  const image = imageUrl || originalUrl || previewUrl;
+  const image = imageUrl || previewUrl || originalUrl;
```

Comment:
sometimes `originalUrl` is a url that Rainbow can't handle, so deprioritizing it in favor of `previewUrl` which is consistently a google cdn url